### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by mattcreaser**
Initial version of the GoRTC library.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    grunt.util.spawn(build, function(err) {
+      if (err) {
+        return done(err);
+      }
+
+      done();
+    });
+  });
+
+  grunt.registerTask('component:build:prod', function() {
```
Comment (colinmacdonald) [inline Gruntfile.js:124]: Don't need the build:prod grunt task

File: Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
+/*jshint node:true*/
+'use strict';
+
+var path = require('path');
+require('js-yaml');
+
+var CLEAN_DIRS = [
+  'components',
+  'dist',
```
Comment (colinmacdonald) [inline Gruntfile.js:9]: dist dir will go away with removing the build:prod task

File: package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "grunt-contrib-clean": "0.5.0",
+    "grunt-contrib-connect": "0.3.0",
+    "grunt-saucelabs": "4.1.2",
+    "lodash": "2.2.1",
+    "gi-assert": "git://github.com/goinstant/assert.git#v1.0.0",
+    "sinon": "~1.7.3",
+    "wrench": "~1.5.1",
+    "winston": "~0.7.2",
+    "underscore.string": "2.3.3",
+    "widget-examples": "git://github.com/goinstant/widget-examples.git"
```
Comment (colinmacdonald) [inline package.json:39]: If there wont be any examples `widget-examples` can be removed.

File: Gruntfile.js (hunk 1 above)
Comment (mattcreaser) [inline Gruntfile.js:124]: We'd still want this for anyone external who wanted to use the library without using component.js no?

Comment (colinmacdonald) [inline Gruntfile.js:124]: Maybe, we just don't make the usercache or colors-common available in that way. The build:prod task was only really there for bower and the CDN.

Comment (colinmacdonald) [inline Gruntfile.js:124]: Maybe we want to make this one available as a bower component then?

**Comment by colinmacdonald** [comment]
LGTM

**Pull request merged**